docs/interactions/Application_Commands.md

```diff
@@ -122,39 +169,208 @@ headers = {
+| -------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------ |
+| id             | snowflake                                                                                                                                                            | the id of the command                            |
+| application_id | snowflake                                                                                                                                                            | the id of the application the command belongs to |
+| guild_id       | snowflake                                                                                                                                                            | the id of the guild                              |
+| permissions    | array of [application command permissions](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-permissions-object-application-command-permissions-structure) | the permissions for the command in the guild     |
+
+###### Application Command Permissions Structure
+
+Application command permissions allow you to enable or disable commands for specific users or roles within a guild.
+
+| Field      | Type                                                                                                                                                      | Description                           |
+| ---------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------- |
+| id         | snowflake                                                                                                                                                 | the id of the role or user            |
+| type       | [application command permission type](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-permissions-object-application-command-permission-type) | role or user                          |
+| permission | boolean                                                                                                                                                   | `true` to allow, `false`, to disallow |
+
+###### Application Command Permission Type
+
+| Name | Value |
+| ---- | ----- |
+| ROLE | 1     |
+| USER | 2     |
+
+Need to keep some of your commands safe from prying eyes, or only available to the right people? Commands support permission overwrites! For both guild _and_ global commands of all types, you can enable or disable a specific user or role in a guild from using a command.
+
+> info
+> For now, if you don't have permission to use a command, they'll show up in the command picker as disabled and unusable. They will **not** be hidden.
```

Comment:
Is this limitation true even for the newfangled context menu commands?

Reply:
yes